assets/i18n/en.json

```diff
@@ -615,6 +619,12 @@
     "I18N_LEARNER_DASHBOARD_INCOMPLETE_SECTION": "In Progress",
     "I18N_LEARNER_DASHBOARD_INTRO_MESSAGE_PART_ONE": "It looks like you haven't tried any of our explorations yet.",
     "I18N_LEARNER_DASHBOARD_INTRO_MESSAGE_PART_TWO": "Let's get started on this exciting journey!",
+    "I18N_LEARNER_DASHBOARD_LEARNER_GROUPS_SECTION": "Learner Groups",
+    "I18N_LEARNER_DASHBOARD_LEARNER_GROUPS_SECTION_GROUPS": "Your Groups",
+    "I18N_LEARNER_DASHBOARD_LEARNER_GROUPS_SECTION_INVITATIONS": "Your Invitations",
+    "I18N_LEARNER_DASHBOARD_LEARNER_GROUPS_SECTION_NO_GROUPS": "You are not part of any groups yet.",
```

Comment:
You haven't joined any learner groups yet

Reply:
done!